Networking/NetworkingTests/Mapper/CustomerMapperTests.swift

```diff
@@ -0,0 +1,87 @@
+import XCTest
+@testable import Networking
+
+/// CustomersMapper Unit Tests
+///
+final class CustomerMapperTests: XCTestCase {
+
+    /// Site ID for testing.
+    private let sampleSiteID: Int64 = 1234
+
+    func test_customer_is_properly_parsed() throws {
+        // Given
+        let jsonData = try XCTUnwrap(Loader.contentsOf("customer"))
+
+        // When
+        let responseCustomer = try CustomerMapper(siteID: sampleSiteID).map(response: jsonData)
+
+        // Then
+        let johnDoe = try sampleCustomer()
+        XCTAssertEqual(responseCustomer, johnDoe)
+    }
+
+    func test_customer_is_correctly_encoded() throws {
+        // Given
+        let johnDoe = try sampleCustomer()
+
+        // When
+        let encodedCustomerData = try CustomerMapper(siteID: sampleSiteID).map(customer: johnDoe)
+        let customerDictionary = try JSONSerialization.jsonObject(with: encodedCustomerData) as? [String: Any]
+
+        // Then
+        XCTAssertEqual(customerDictionary?["first_name"] as? String, "John")
+        let billingAddressDictionary = try XCTUnwrap(customerDictionary?["billing"] as? [String: Any])
+        XCTAssertEqual(billingAddressDictionary["first_name"] as? String, "John")
+    }
```

Comment:
I've tried to make full round-trip on `Customer` mapping here, but Xcode said NO 😑

<img width="940" alt="image" src="https://user-images.githubusercontent.com/3132438/102533386-6654ef80-40b6-11eb-962a-c314959c78b1.png">

Reply:
LoL anyway, is not needed to use JSONSerialization directly here 👍